Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM instrumentisto/dep:0.4.1 AS builder
```

Comment:
better use official golang Go images. That way someone reading this dockerfile doesn't have to go figure out what instrumentisto images do. The build process is simple enough to be able to use a vanilla approach.